src/convertToObject.js

```diff
@@ -6,7 +6,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .trim()
+    .split(';')
+    .map((declaration) => declaration.split(':'))
```

Comment:
The `map` function splits each declaration into a property and value pair. Ensure that the input string is well-formed, as this could lead to unexpected results if the input is not formatted correctly.